src/main/webapp/editFilesFragment.xhtml

```diff
@@ -155,6 +160,7 @@
                                   fileLimit="#{EditDatafilesPage.getMaxNumberOfFiles()}" 
                                   invalidSizeMessage="#{bundle['file.edit.error.file_exceeds_limit']}" 
                                   sequential="true"
+                                  previewWidth="-1"
```

Comment:
what does this do?

Reply:
`TL;DR` it disables the browser-side thumbnail previews. 

I was delighted to discover that, as of a couple of recent versions of PrimeFaces, it is now possible to disable their own image thumbnail preview in the upload component. It used to drive me nuts - we don't really need it (because we generate our own thumbnails server-side); but, unlike with our thumbnails, there's no way to define a size limit, so attempting to upload a bunch of large enough images paralyzes the user's browser (since it's all javascript). And, adding insult to injury, the user assumes that it's our site that's crapping the bed... I am embarrassed to acknowledge that, despite all that righteous indignation, I somehow missed the point at which they had added this ability to disable the whole thing, by setting the previewWidth to -1.